source/user-manual/capabilities/file-integrity/fim-configuration.rst

```diff
@@ -448,17 +448,14 @@ The main advantage of using in memory database is the performance as reading and
 Configuring synchronization
 ---------------------------
 
-:ref:`Synchronization <reference_ossec_syscheck_synchronization>` can be configured to change the synchronization interval, the number of events per second, the queue size and the response timeout:
+You can configure :ref:`synchronization <reference_ossec_syscheck_synchronization>` to change the synchronization interval and the maximum number of events per second:
```

Comment:
You can also configure the number of events per second, the queue size, and the response timeout. They have been removed from the config template but the settings are available.

We don't add these settings to the `<synchronization>` block in examples because 99% of users won't need to modify them, and they add complexity.